src/dom/Draggable.js

```diff
@@ -112,7 +112,7 @@ export var Draggable = Evented.extend({
 		// Fired when a drag is about to start.
 		this.fire('down');
 
-		var first = e.touches ? e.touches[0] : e,
+		var first = e.touches && e.touches.length > 0 ? e.touches[0] : e,

```

Comment:
In this line we are handling `touchstart` event.
And it MUST https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/touches property.

And if you find that it doesn't - then we should find we that ever happened.

Never try to fix issues blindly.

Reply:
Fair enough.  Thank you for your insight.